godtossing/Pos/Ssc/GodTossing/Toss/Pure.hs

```diff
@@ -39,11 +39,8 @@ type MultiRichmenSet   = HashMap EpochIndex RichmenSet
 -- require randomness even though they are deterministic. Note that running
 -- them with the same seed every time is insecure and must not be done.
 newtype PureToss a = PureToss
-    { getPureToss :: NamedPureLogger (
-                     StateT GtGlobalState (
-                     Rand.MonadPseudoRandom Rand.ChaChaDRG)) a
-    } deriving (Functor, Applicative, Monad,
-                CanLog, HasLoggerName, Rand.MonadRandom)
+    { getPureToss :: StateT GtGlobalState (NamedPureLogger (Rand.MonadPseudoRandom Rand.ChaChaDRG)) a
+    } deriving (Functor, Applicative, Monad, MonadState GtGlobalState, CanLog, HasLoggerName, Rand.MonadRandom)
```

Comment:
We use 80 characters as line length limit, in some cases it can be 100 if necessary, but this line has 111 characters, so I don't get why you changed `deriving` clause formatting here.
Cosmetic, but still.